packages/react-examples/README.md

```diff
@@ -1,47 +1,77 @@
 # @fluentui/react-examples
 
-**Examples and documentation for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+This package contains examples and documentation site content for `@fluentui/react` v8 and related packages, including `@fluentui/react-charting` and `@fluentui/react-experiments`.
 
-This package is only meant to be consumed by variants of the Fluent UI documentation site and therefore has **NO GUARANTEE OF API STABILITY**.
+Each folder under `src` is the unscoped name of a package, containing examples and documentation content for that package. For example, content for `@fluentui/react` is under `src/react`.
 
-## Architecture
+The examples under `react` (plus `react-focus`) are used on the [public documentation site](https://developer.microsoft.com/fluentui) via `apps/public-docsite`, and [aka.ms/fluentdemo](https://aka.ms/fluentdemo) via `apps/public-docsite-resources`.
+
+The other packages' examples are only used for local development and on the PR deploy site ([here for `master`](https://fluentuipr.z22.web.core.windows.net/heads/master/)).
+
+## Background
+
+Examples were originally moved here to resolve circular dependency issues back when we were trying to split `@fluentui/react` into smaller component packages before the v8 release, and during early development of v9 prior to changes in the typescript configuration. (This was with a typescript configuration with strict dependency declaration and package boundaries, rather path resolutions that allow examples/stories to import from any component package even if it's not a dependency.) For example, the goal was to allow `Button`'s examples to use `Toggle` without introducing a dependency on the toggle package in the button package.
+
+Since we mostly didn't end up splitting packages in v8, and went with a different approach for v9, the only package where the circular dep issue is relevant now is `@fluentui/react-focus`. Otherwise, the examples could theoretically be moved back into component packages.
+
+## Local development
```

Comment:
I felt like this should be explicitly mentioned somewhere in the docs (the fact that you don't start the stories by running commands within this package)